magento/magento2
# #22047 Feature: Newrelic transaction name based on CLI name

### Description (*)
<img width="1447" alt="Screenshot 2019-03-30 at 13 51 23" src="https://user-images.githubusercontent.com/1639941/55276325-ef306400-52f2-11e9-85f0-d35311a61d08.png">


### Fixed Issues (if relevant)

1. magento/magento2#22047: Magento CRON Job Names are missing in NewRelic: "Transaction Names"

### Manual testing scenarios (*)
1. Configure your Newrelic setup
2. Run any CLI command
3. Check transaction list

### Contribution checklist (*)
